jobs/sentry-error-loader/sentry_error_loader.py

```diff
@@ -55,27 +56,28 @@ def fetch_and_clean_from_clickhouse(project_slug, target_date):
                                    WHERE toDate(timestamp) == '{target_date}'"""
     )
 
-    cleaned_df = all_rows.rename(make_name_bq_safe, axis="columns")
-
-    cols_with_nulls_in_arrays = [
-        "exception_frames_colno",
-        "exception_frames_package",
-        "exception_stacks_mechanism_type",
-        "exception_stacks_mechanism_handled",
-    ]
-    for col_name in cols_with_nulls_in_arrays:
-        cleaned_df[col_name] = cleaned_df[col_name].apply(process_arrays_for_nulls)
-
-    """
-    Pandas' default timestamp datatype returned by clickhouse-connect writes out unix timestamps
-    in nanoseconds, but they get interpreted by BQ as microseconds. A quick cast to datetime string
-    before writing avoids triggering the issue.
-    """
-    cols_with_unix_timestamps = ["timestamp", "message_timestamp", "received"]
-    for col_name in cols_with_unix_timestamps:
-        cleaned_df[col_name] = cleaned_df[col_name].apply(str)
-
-    extract.data = cleaned_df
+    if not all_rows.empty:
```

Comment:
nit: this might be better to flip around, so it's a [guard clause](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html)

Reply:
Addressed in [8650185](https://github.com/cal-itp/data-infra/pull/2356/commits/86501859f71672b832bbab6f639a22064c08479d)